pkgs/development/libraries/gnu-efi/default.nix

```diff
@@ -15,14 +17,9 @@ stdenv.mkDerivation rec {
 
   makeFlags = [
     "PREFIX=\${out}"
-    "CC=${stdenv.cc.targetPrefix}gcc"
-    "AS=${stdenv.cc.targetPrefix}as"
-    "LD=${stdenv.cc.targetPrefix}ld"
-    "AR=${stdenv.cc.targetPrefix}ar"
-    "RANLIB=${stdenv.cc.targetPrefix}ranlib"
-    "OBJCOPY=${stdenv.cc.targetPrefix}objcopy"
-  ] ++ stdenv.lib.optional stdenv.isAarch32 "ARCH=arm"
-    ++ stdenv.lib.optional stdenv.isAarch64 "ARCH=aarch64";
+    "HOSTCC=${buildPackages.stdenv.cc.targetPrefix}cc"
```

Comment:
We mostly assume that the "build" machine's compiler (HOSTCC to gnu-efi) is just "cc", but perhaps we shouldn't in some more complicated instances. /cc @Ericson2314

Reply:
@matthewbauer Yeah I think it's just more consistent to never assume. Also, I am tempted to drop specific arch processing and do `build-cc` `cc` `target-cc`, once we can get rid of the infix salt :D.